api/src/common/components.py

```diff
@@ -24,6 +24,16 @@
         "alt_name": "Mercury",
         "molecular_weight": 200.59,
     },
+    "6": {
+        "chemical_formula": "MEG",
+        "alt_name": "ethane-1-2-diol",
```

Comment:
From wikipedia, the formula looks to be "C2H4(OH)2", and the "alt_name" 'MEG' or 'monoetylenglykol'.
Where did you find this?
Either way, they should be swtiched around.